src/components/Kv/KvAccordionItem.vue

```diff
@@ -1,30 +1,26 @@
 <template>
 	<div
-		class="kv-accordion"
-		:class="{
-			'kv-accordion--open' : isOpen,
-		}"
+		class="tw-border-b tw-border-tertiary tw-relative"
```

Comment:
To prevent a bottom border on the last entry the previous CSS used last-child. To bring that back with Tailwind, you can use `last:tw-border-b-0`

Reply:
Good point.
Class is included now.